credentials/apps/credentials/models.py

```diff
@@ -198,7 +198,7 @@ def course_key(self):
 OrganizationDetails = namedtuple('OrganizationDetails', ('uuid', 'key', 'name', 'display_name',
                                                          'certificate_logo_image_url'))
 ProgramDetails = namedtuple('ProgramDetails', ('uuid', 'title', 'subtitle', 'type', 'course_count',
-                                               'organizations', 'hours_of_effort'))
+                                               'organizations', 'hours_of_effort', 'courses'))
```

Comment:
Can we please introduce and use philu switch here, to avoid core change in unit tests?

Reply:
For now all the unit tests are passed, So I think we can avoid waffle switch here as it increase the core changes.